packages/tasks/src/model-libraries.ts

```diff
@@ -715,6 +715,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path:"model_vae_fp16.pt"`,
 		snippets: snippets.threedtopia_xl,
 	},
+	cotracker: {
+		prettyLabel: "CoTracker",
+		repoName: "CoTracker",
+		repoUrl: "https://github.com/facebookresearch/co-tracker",
+		filter: false,
+		countDownloads: `path_extension:"pt"`,
```

Comment:
```suggestion
		countDownloads: `path_extension:"pth"`,
```

I just noticed the file extension is not `pt` but `pth` in https://huggingface.co/facebook/cotracker/tree/main.